modules/Bio/EnsEMBL/Compara/PipeConfig/PairAligner_conf.pm

```diff
@@ -642,47 +630,72 @@ sub core_pipeline_analyses {
               -analysis_capacity => 1,
               -rc_name => '8Gb_job',
           },
-	    { -logic_name => 'healthcheck',
-	      -module => 'Bio::EnsEMBL::Compara::RunnableDB::HealthCheck',
-	      -parameters => {
-			      'previous_db' => $self->o('previous_db'),
-			      'ensembl_release' => $self->o('ensembl_release'),
-			      'prev_release' => $self->o('prev_release'),
-			      'max_percent_diff' => $self->o('patch_alignments') ? $self->o('max_percent_diff_patches') : $self->o('max_percent_diff'),
-			     },
-	      -wait_for => [ 'set_internal_ids_collection' ],
-	      -rc_name => '1Gb_job',
-	    },
-	    { -logic_name => 'pairaligner_stats',
-	      -module => 'Bio::EnsEMBL::Compara::RunnableDB::PairAligner::PairAlignerStats',
-	      -parameters => {
-			      # 'skip' => $self->o('skip_pairaligner_stats'),
-            'skip' => '#expr( #skip_pairaligner_stats# || #patch_alignments# )expr#',
-			      'dump_features' => $self->o('dump_features_exe'),
-			      'compare_beds' => $self->o('compare_beds_exe'),
-			      'create_pair_aligner_page' => $self->o('create_pair_aligner_page_exe'),
-			      'bed_dir' => $self->o('bed_dir'),
-			      'ensembl_release' => $self->o('ensembl_release'),
-			      'reg_conf' => $self->o('reg_conf'),
-			      'output_dir' => $self->o('output_dir'),
-			     },
-	      -wait_for =>  [ 'healthcheck' ],
-              -flow_into => {
-                  'A->1' => [ 'coding_exon_stats_summary' ],
-                  '2->A' => [ 'coding_exon_stats' ],
-			     },
-	      -rc_name => '2Gb_job',
-	    },
-            {   -logic_name => 'coding_exon_stats',
-                -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PairAligner::PairAlignerCodingExonStats',
-                -hive_capacity => 5,
-                -rc_name => '2Gb_job',
+
+        {   -logic_name => 'detect_component_mlsss',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PairAligner::DetectComponentMLSSs',
+            -parameters => {
+                'do_pairwise_gabs'          => $self->o('do_pairwise_gabs'),
+                'do_compare_to_previous_db' => $self->o('do_compare_to_previous_db'),
             },
-            {   -logic_name => 'coding_exon_stats_summary',
-                -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PairAligner::PairAlignerCodingExonSummary',
-                -rc_name => '1Gb_job',
+            -wait_for   => [ 'set_internal_ids_collection' ],
```

Comment:
I know, but the purpose of this PR is not to update the whole pipeline (which is a huge and not-easy task). So I have left the connection to the first analysis of "the new part" as it was (from the old `healthcheck`), and used semaphores from there on.

Reply:
I have left it only as it was the connection to the previous analysis of this block. To update this further it would require a whole pipeline update, which is a mammoth (new compara-favourite word!) task and not the objective of this PR. I have used semaphores in the new additions to the pipeline.

Reply:
I have left it only as it was the connection to the previous analysis of this block. To update this further it would require a whole pipeline update, which is a mammoth (new compara-favourite word!) task and not the objective of this PR. I have used semaphores in the new additions to the pipeline.

Reply:
Fair enough. I didn't expect that you'd remove this from the whole pipeline, just not introduce new cases.